GCEWindowsAgent/oslogin.go

```diff
@@ -0,0 +1,339 @@
+//  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
os_login.go may make more sense, but I don't mind of you keep this as oslogin.go